src/http/index.js

```diff
@@ -29,6 +29,24 @@ function hapiInfoToMultiaddr (info) {
   return toMultiaddr(uri)
 }
 
+async function serverCreator (serverAddrsArr, createServerFunc, hapiInfoToMultiaddr, ipfs) {
```

Comment:
Please can we rename `createServerFunc` to `createServer`? By convention we don't add type information to our variable names in this project.